src/libraries/System.IO.Packaging/tests/Tests.cs

```diff
@@ -181,6 +181,27 @@ public void PackageOpen_Open_EmptyStream_Throws()
             }
         }
 
+        [Fact]
+        public void PackageOpen_Open_InvalidContent_Throws()
+        {
+            var temp = GetTempFileInfoWithExtension(".docx").FullName;
+
+            using (FileStream fs = File.OpenWrite(temp))
+            {
+                byte[] bytes = File.ReadAllBytes("plain.docx");
+                fs.Write(bytes, 0, bytes.Length);
+                fs.Seek(500, SeekOrigin.Begin);
+                fs.Write(new byte[500], 0, 500); // garble it
```

Comment:
```suggestion
                byte[] bytes = File.ReadAllBytes("plain.docx");
                bytes.Slice(500, 500).Clear();
                fs.Write(bytes, 0, bytes.Length);
```
?

Reply:
Yes, neater. I forgot we have spans on .NET Framework.